src/cil.ml

```diff
@@ -1844,9 +1852,11 @@ let d_const () c =
          FFloat -> chr 'f'
        | FDouble -> nil
        | FLongDouble -> chr 'L'
+       | FFloat128 -> chr 'L' (* TODO: correct? *)
```

Comment:
https://gcc.gnu.org/onlinedocs/gcc/Floating-Types.html says:

> Use a suffix ‘q’ or ‘Q’ for `__float128`.

Reply:
Nice catch, `L` is wrong here for sure!